lib/fog/ovirt/requests/compute/v4/update_vm.rb

```diff
@@ -9,7 +9,7 @@ def update_vm(attrs)
             vm_service = client.system_service.vms_service.vm_service(attrs[:id])
 
             if attrs[:cores].present?
-              cpu_topology = OvirtSDK4::CpuTopology.new(:cores => attrs[:cores], :sockets => "1")
+              cpu_topology = OvirtSDK4::CpuTopology.new(:cores => attrs[:cores], :sockets => attrs[:sockets] || 1)
```

Comment:
The type of the default value is changing from string to int, is it intentional?